ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -46,14 +46,18 @@ const EditGasItem = ({ priorityLevel }) => {
     maxFeePerGas = gasFeeEstimates[priorityLevel].suggestedMaxFeePerGas;
     maxPriorityFeePerGas =
       gasFeeEstimates[priorityLevel].suggestedMaxPriorityFeePerGas;
-  } else if (
-    priorityLevel === PRIORITY_LEVELS.DAPP_SUGGESTED &&
-    dappSuggestedGasFees
-  ) {
-    maxFeePerGas = hexWEIToDecGWEI(dappSuggestedGasFees.maxFeePerGas);
-    maxPriorityFeePerGas = hexWEIToDecGWEI(
-      dappSuggestedGasFees.maxPriorityFeePerGas,
-    );
+  } else if (priorityLevel === PRIORITY_LEVELS.DAPP_SUGGESTED) {
+    if (estimateUsed === PRIORITY_LEVELS.DAPP_SUGGESTED) {
+      maxFeePerGas = maxFeePerGasValue;
+      maxPriorityFeePerGas = maxPriorityFeePerGasValue;
+    } else if (dappSuggestedGasFees) {
```

Comment:
Could you explain why this change was made? Why do we want to select the values stored in the `gasFeeContext` over the `dappSuggestedFees` on the `transaction` object?

Reply:
In V2 implementation values from `useGasFeeContext()` are dapp suggested if estimateUsed is `dappSuggested`.

Reply:
I anyways updated this to always use `dappSuggestedGasFees` from `transaction`.